.travis.yml

```diff
@@ -24,7 +24,7 @@ before_script:
     - cd $TRAVIS_BUILD_DIR
     - if astyle --style=stroustrup --indent-switches --indent-labels --pad-oper --pad-header --align-pointer=name --add-brackets --convert-tabs --max-code-length=90 --break-after-logical --suffix=none *.c *.h --recursive --exclude=astyle --exclude=src/yajl --exclude=src/secp256k1 --exclude=tests/windows/hidapi --dry-run -Q | grep "Formatted" ; then exit 1 ; fi;
     - gpg --import contrib/contributors_gpg_keys/*
-    - if git log --pretty="format:%H   %aN   %s   %G?" f0a012e224fedac87b0393a161d4af1473fea74a..HEAD^1 | (grep -v "${t} G$" || grep -v "${t} U$"); then exit 1 ; fi;
+    - if git log --pretty="format:%H   %aN   %s   %G?" f0a012e224fedac87b0393a161d4af1473fea74a..HEAD^1 | grep -v "${t} G$" | grep -v "${t} U$"; then exit 1 ; fi;
```

Comment:
I had to make a small change here. The second grep for an ending `U` stopped occurring. Strange as it worked for #66.